src/authService.js

```diff
@@ -17,22 +18,41 @@ export class AuthService {
 
   getMe(criteria) {
     if (typeof criteria === 'string' || typeof criteria === 'number') {
-      criteria = {id: criteria};
+      criteria = { id: criteria };
     }
     return this.client.find(this.auth.getProfileUrl(), criteria);
   }
 
+  getCurrentToken() {
+    return this.auth.getToken();
+  }
+
+  getRefreshToken() {
+    return this.auth.getRefreshToken();
+  }
+
   updateMe(body, criteria) {
     if (typeof criteria === 'string' || typeof criteria === 'number') {
-      criteria = {id: criteria};
+      criteria = { id: criteria };
     }
     return this.client.update(this.auth.getProfileUrl(), criteria, body);
   }
 
   isAuthenticated() {
+    let isExpired = this.auth.isTokenExpired();
+    if (isExpired && this.config.autoUpdateToken) {
```

Comment:
No it'll return the value of isAuthenticated() from the Authentication Class not here